spec/jobs/open_prison_transfer_job_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe OpenPrisonTransferJob, type: :job do
+  include ActiveJob::TestHelper
+
+  let(:nomis_offender_id) { 'G3462VT' }
+  let(:nomis_staff_id) { 485_637 }
+  let(:open_prison_code) { 'HDI' }
+  let(:movement_json) {
+    {
+      offenderNo: nomis_offender_id,
+      fromAgency: "LEI",
+      toAgency: open_prison_code,
+      movementType: "TRN",
+      directionCode: "IN"
+    }.to_json
+  }
+  let(:elite2api) { 'https://gateway.t3.nomis-api.hmpps.dsd.io/elite2api/api' }
+
+  before do
+    stub_auth_token
+  end
+
+  it 'does not send an email when no LDU email address' do
+    allow(OffenderService).to receive(:get_offender).and_return(Nomis::Offender.new.tap{ |o|
+      o.prison_id = open_prison_code
+      o.offender_no =  nomis_offender_id
+      o.case_allocation = 'NPS'
+    })
+
+    stub_request(:get, "#{elite2api}/staff/#{nomis_staff_id}/emails").
```

Comment:
There's a 'stub_poms' method in ApiHelper - IMHO we should be adding to that rather than doing 'naked' API stubbing